js/player.js

```diff
@@ -0,0 +1,9 @@
+//A player object to store game info
+let player = {
+  hp: 100,
+  atk: 5,
+  hasShield: false,
```

Comment:
this is good for now I suppose as this might be the only effect we'll have for the player at the end of this sprint, however, I feel like it might be more beneficial some point in the future if we have an array of states we can add to the player instead of having a property for each effect in player. This will allow us to stack effects and mix effects without having to adjust multiple properties at once.